Polyhedron/demo/Polyhedron/Scene_spheres_item.h

```diff
@@ -0,0 +1,97 @@
+#ifndef SCENE_SPHERES_ITEM_H
+#define SCENE_SPHERES_ITEM_H
+#include "Scene_basic_objects_config.h"
+#include "create_sphere.h"
+#include "CGAL/Three/Scene_group_item.h"
+#include <CGAL/Three/Scene_item.h>
+#include <CGAL/Three/Scene_interface.h>
+#include <CGAL/Three/Viewer_interface.h>
+#include <CGAL/Sphere_3.h>
+#include <CGAL/Plane_3.h>
+#include <CGAL/Exact_predicates_inexact_constructions_kernel.h>
+#include <QOpenGLShaderProgram>
+
+#include <QList>
+#include<vector>
+
+class SCENE_BASIC_OBJECTS_EXPORT Scene_spheres_item
+    : public CGAL::Three::Scene_item
+{
+  Q_OBJECT
+public:
+  typedef CGAL::Exact_predicates_inexact_constructions_kernel Kernel;
+  typedef std::pair<CGAL::Sphere_3<Kernel>*, CGAL::Color> sphere_pair ;
```

Comment:
type names usually start with a capitalized letter in CGAL.
`sphere_pair` should be `Sphere_pair`